README.md

```diff
@@ -1,8 +1,15 @@
-# Commerce BTCPay
+# Accept Bitcoin on your Drupal Store using BTCPay Server
```

Comment:
There are different Drupal ecommerce solution, the one my plugin is for is called "Drupal Commerce" so I would suggest to change "Drupal Store" to "Drupal Commerce store"